src/resources/resources.cpp

```diff
@@ -58,7 +58,7 @@ QString brandThemePath()
 
 bool Resources::isVanillaTheme()
 {
-    return std::string_view(APPLICATION_SHORTNAME) == "OpenCloud";
+    return std::string_view(APPLICATION_SHORTNAME).starts_with("OpenCloud");
```

Comment:
This seems to cry for trouble as we move forward... Why not set a flag in CMake?